test/views/CalculationViewSpec.scala

```diff
@@ -105,11 +104,10 @@ class CalculationViewSpec extends SpecBase with ViewBehaviours with ViewUtils {
     "if it is an additional penalty and the penalty is not estimated" must {
```

Comment:
Similarly here "if it is an additional penalty and the penalty is not estimated" can be "if it is second late payment penalty and the penalty is not estimated"